src/openrpc/discovery.json

```diff
@@ -971,7 +971,7 @@
           ]
         }
       ],
-      "summary": "Launch or foreground the specified app, and optionally instructs it to navigate to the specified user action",
+      "summary": "Launch or foreground the specified app, and optionally instructs it to navigate to the specified user action. \n For the Primary Experience, the appId can any one of <br/> urn:firebolt:apps:main <br/> xrn:firebolt:application-type:main <br/> or xrn:firebolt:application-type:settings",
```

Comment:
We should not document `urn:firebolt:apps:main` as it's not to spec.

Reply:
And i think you're missing a word, maybe "For the Primary Experience, the appId can **be** any one of:"

Also, it's best to avoid HTML in markdown. Try using \n\n and putting the values in a list with ` - ` in front of each.